packages/ui/src/components/vuestic-components/va-tabs/VaTab/VaTab.vue

```diff
@@ -15,7 +15,7 @@
       class="va-tab__content"
       v-on="context.keyboardFocusListeners"
       @focus="onFocus"
```

Comment:
When you size down the window - tab title jumps right and left.
![image](https://user-images.githubusercontent.com/5394573/123817762-1b3b5500-d901-11eb-8045-872c9dafa440.png)

Not a huge deal. You can move to different issue or something.

Reply:
Fixed it. The bug occurred when there was only one tab in the tab container.